recipes/krakenuniq/build.sh

```diff
@@ -1,6 +1,24 @@
 #!/bin/bash
+export CPLUS_INCLUDE_PATH=${CPLUS_INCLUDE_PATH}:${PREFIX}/include
+export LIBRARY_PATH="${PREFIX}/lib"
+export LD_LIBRARY_PATH="${PREFIX}/lib"
+
 mkdir -p "$PREFIX/libexec" "$PREFIX/bin"
 
+# src/Makefile uses
+# > CXX = g++
+# > CXXFLAGS = -Wall -fopenmp -O3
+# We patch these to add to environment, rather than override
+# because we can't get to the `make` call easily.
+sed -i.bak 's/CXX *=/CXX ?=/; s/CXXFLAGS *=/CXXFLAGS +=/' src/Makefile 
```

Comment:
I'm not a big fan of running `sed`, etc. as part of the build script.  It can be non-deterministic, and cause unexpected changes if the file being edited changes later.  I always try to create a patch file instead whenever possible.

Reply:
I totally agree. I was a little bit lazy and just transferred the kraken1 recipe but I will create a patch file.

Reply:
Sure, no worries.